data/emulator.js

```diff
@@ -4060,8 +4060,29 @@ class EmulatorJS {
         
         if (this.gameManager.getDiskCount() > 1) {
             const diskLabels = {};
+            let isM3U = false;
+            let disks = {};
+            if (this.fileName.split(".").pop() == "m3u") {
```

Comment:
In JavaScript, we should use 3 equal signs. Why JavaScript does this? Who knows

Reply:
I looked it up ```===``` is a strict equality. You need to do this in JS because JS isn't strongly typed like c# or other languages are.

Reply:
That's it, I can never actually remember what it means so I just never use == in JavaScript